_posts/2022-08-23-New-series-From-the-editor's-desk.md

```diff
@@ -0,0 +1,71 @@
+
+When developing OpenSearch documentation, we strive to create content that is inclusive and free of bias. We use inclusive language to connect with the diverse and global OpenSearch audience, and we are careful in our word choices. Inclusive and bias-free content improves clarity and accessibility of our content for all audiences, so we avoid ableist and sexist language and language that perpetuates racist structures or stereotypes. In practical terms, this means that we do not allow certain terms to appear in our content, and we avoid using others, depending on the context.
+
+## Inclusive terminology
+
+The following tables list offensive terms that we do not use in our content and sensitive terms that we avoid using. 
+
+We do not use the following terms in OpenSearch content. Instead, use one of the acceptable alternatives provided in the second column.
+
+| Don’t use      | Use instead                 |
+|----------------|-----------------------------|
+| abort          | stop                        |
+| black day      | blocked day                 |
+| blacklist      | deny list                   |
+| execute        | start, run                  |
+| hang           | stop responding             |
+| kill           | end, stop                   |
+| master         | primary, main, leader       |
+| master account | management account          |
+| slave          | replica, secondary, standby |
+| white day      | open day                    |
+| whitelist      | allow list                  |
+
+We avoid using the following terms in OpenSearch content, depending on the context. Again, you can use one of the alternatives listed in the second column.
+
+| Avoid using              | Use instead                         |
+|--------------------------|-------------------------------------|
+| blackout                 | service outage, blocked             |
+| demilitarized zone (DMZ) | perimeter network, perimeter zone   |
+| disable                  | turn off, deactivate, stop          |
+| enable                   | turn on, activate, start            |
+| invalid                  | not valid                           |
+| primitive                | primitive data type, primitive type |
+| purge                    | delete, clear, remove               |
+| segregate                | separate, isolate                   |
+| trigger                  | initiate, invoke, launch, start     |
+| white day                | open day                            |
+| whitelist                | allow list                          |
+
+Version 2.0 replaced non-inclusive terminology (such as master, blacklist) throughout OpenSearch with inclusive terminology (such as cluster manager, allow list), but please let us know if you think we missed something so that we can address it in an expedient manner. Check out the associated [GitHub issue](https://github.com/opensearch-project/OpenSearch/issues/2589) for more information.
```

Comment:
Consider backlinking to release blog when referencing Version 2.0? It references this update. https://opensearch.org/blog/releases/2022/05/opensearch-2-0-is-now-available/